### PR TITLE
Navbar

### PR DESCRIPTION
**Pull request by guilherme-gjv**
# Related Issue
- #73 

# Proposed Changes
- New font families implemented;
- Darker navbar;
- Hexagon icon design implemented;

# Question
- Should I keep this bell icon?

# Checklist
- [ ] Tests
- [ ] Translations
- [ ] Documentation

# Screenshots
>## PC
> navbar

>## Mobile
> navbarm
>
> navbarmo

### CODE REVIEW

**Review by Manuel-Antunes — CHANGES_REQUESTED** [review]
there's a reference of `Cinzel` in fonts, but, it isn't imported in anywhere and the hexagon isn't good

**Review by leticiabsduarte — CHANGES_REQUESTED** [review]
The font needs to be imported, please correct it.
Improve the hexagon icon, try to make it more like the original ideia

**Comment by guilherme-gjv** [comment]
tempsnip

**Comment by guilherme-gjv** [comment]
# 👌 

sim

File: resources/views/layouts/main.edge [hunk 1]
```diff
@@ -6,6 +6,8 @@
   <meta http-equiv="X-UA-Compatible" content="ie=edge">
   <link href="/images/favicon.ico" rel="icon" type="image/ico">
   <link href="https://fonts.googleapis.com/css?family=Montserrat:300,700" rel="stylesheet">
+  <link href="https://fonts.googleapis.com/css?family=Cinzel:400" rel="stylesheet">
+  <link href="https://fonts.googleapis.com/css?family=Roboto:300,400" rel="stylesheet">
```
Comment (Manuel-Antunes) [inline resources/views/layouts/main.edge:10]:
```suggestion
  <link href="https://fonts.googleapis.com/css?family=Cinzel:400" rel="stylesheet">
  <link href="https://fonts.googleapis.com/css?family=Roboto:300,400" rel="stylesheet">
```

please, import font in app.css

**Review by Manuel-Antunes — APPROVED** [review]
Please, import fonts in css instead of html

**Review by leticiabsduarte — APPROVED** [review]
The user icon is perfect, but now put the notification dot, it's important. Good job =)

**Pull request merged**